src/assisted_test_infra/test_infra/controllers/tang_controller/tang_controller.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+import socket
+
+import consts
+from assisted_test_infra.test_infra import utils
+from service_client import log
+
+
+class TangController:
+    def __init__(
+            self,
+            name: srt = None,
+            port: int = consts.DEFAULT_TANG_SERVER_PORT,
+            pull_secret: str = None
+    ):
+        self._name = name
+        self._port = port
+        self._pull_secret = pull_secret
+        self._image = "registry.redhat.io/rhel8/tang"
+        self._set_server_address()
+        self._run_tang_server()
+        self._set_thumbprint()
```

Comment:
constructors usually do no complex actions than just copying/setting member values, so I suggest having ``run_tang_server`` as a public method to be used for running the actual server